web/src/components/DiagramSwitcher/DiagramSwitcher.tsx

```diff
@@ -9,18 +10,24 @@ interface IProps {
   selectedType: SupportedDiagrams;
 }
 
+const color = '#FFFFFF';
```

Comment:
Let's think about incorporating the style components theme so we don't have to hardcode colors moving forward (future work)